site/en/docs/webstore/best_practices/index.md

```diff
@@ -9,10 +9,33 @@ description: >
 This page has advice on how you should implement your app and list it in the store. As the store
 matures and we learn from developers' experiences, these recommendations will be updated.
 
-## Design a great app
+## Design a high-quality extension
+
+We encourage you to develop extensions that are of high quality. High-quality extensions conform to
+standards of performance, security, and user experience, as summarized by the following guidelines:
+
+* **Compliance:** Does the extension comply with our [program policies]? Extensions must not violate
+any of these policies.
```

Comment:
That's a fancy bit of shorthand. I didn't know we could write links this way 😳

Reply:
It's concise, but it leaves you with constraints like no uppercase...